java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/Identifier.java

```diff
@@ -0,0 +1,13 @@
+package org.cloudburstmc.protocol.java.data;
+
+import lombok.Value;
+
+@Value
+public class Identifier {
```

Comment:
Use Kyori's `Key` instead.

Reply:
I think I should have addressed this correctly?